liquidator/test/common.js

```diff
@@ -0,0 +1,45 @@
+const Token = artifacts.require("ExpandedERC20");
+
+const { toBN } = web3.utils;
+
+const CONSTANTS = {
+  ETH_ADDRESS: "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE"
```

Comment:
So this is only the ETH_ADDRESS that is used for 1inch contracts? on mainnetfork?

Reply:
Yeah, I've made it so the mock contracts recognize it as well